src/main/java/com/flowingcode/vaadin/addons/twincolgrid/TwinColGrid.java

```diff
@@ -294,6 +295,8 @@ public TwinColGrid(@NonNull Grid<T> availableGrid, @NonNull Grid<T> selectionGri
           side.layout.setSpacing(false);
         });
 
+    setMoveItemsByDoubleClick(true);

```

Comment:
Is it okay that the feature is by default set to true? it will change the current behavior of the component. I think it should be false by default.

Reply:
In the current version if you double click on an item nothing will happen, so I don't think that would change things in a negative way by enabling it by default. If new features don't break existing functionality, I think they should be enabled when added.

Reply:
We had an internal discussion and it could cause problems to devs that are already adding a double click listener for achieving the same behavior, so we will add it to be disabled by default in the current version, and it will be enabled by default in the new upcoming new major version